crates/cli/src/config.rs

```diff
@@ -367,6 +390,41 @@ Fetch the server's fingerprint with:
         self.spacetimedb_token = None;
     }
 }
+impl From<&DocumentMut> for RawConfig {
+    fn from(value: &DocumentMut) -> Self {
+        let default_server = value.get("default_server").and_then(Item::as_str).map(String::from);
+        let web_session_token = value.get("web_session_token").and_then(Item::as_str).map(String::from);
+        let spacetimedb_token = value.get("spacetimedb_token").and_then(Item::as_str).map(String::from);
+
+        let server_configs = value
+            .get("server_configs")
+            .and_then(Item::as_array_of_tables)
+            .map(|arr| {
+                arr.iter()
+                    .map(|table| {
+                        let nickname = table.get("nickname").and_then(Item::as_str).map(String::from);
+                        let host = table.get("host").and_then(Item::as_str).map(String::from).unwrap();
+                        let protocol = table.get("protocol").and_then(Item::as_str).map(String::from).unwrap();
+                        let ecdsa_public_key = table.get("ecdsa_public_key").and_then(Item::as_str).map(String::from);
+                        ServerConfig {
+                            nickname,
+                            host,
+                            protocol,
+                            ecdsa_public_key,
+                        }
```

Comment:
nit: Should we have a `ServerConfig::from_table`?

Reply:
```rust
    pub fn from_table:(table: &Table) -> Self {
        let nickname = table.get("nickname").and_then(Item::as_str).map(String::from);
        let host = table.get("host").and_then(Item::as_str).map(String::from).unwrap();
        let protocol = table.get("protocol").and_then(Item::as_str).map(String::from).unwrap();
        let ecdsa_public_key = table.get("ecdsa_public_key").and_then(Item::as_str).map(String::from);
        ServerConfig {
            nickname,
            host,
            protocol,
            ecdsa_public_key,
        }
    }
```

Reply:
Done